api/v1alpha1/stage_types.go

```diff
@@ -726,17 +726,19 @@ type AnalysisRunArgument struct {
 // VerificationInfo contains information about the currently running
 // Verification process.
 type VerificationInfo struct {
+	// ID is the identifier of the Verification process.
+	ID string `json:"id,omitempty" protobuf:"bytes,1,opt,name=id"`
```

Comment:
@krancour do we have any rules around (not) restructuring the protobuf order while we are still `< v1.0.0`?

Reply:
Not really at this stage, although I've occasionally done it the right way instead of renumbering.

Buuuuut... that was before #1515 

Did you renumber these by hand?

I believe that _since_ #1515, you can omit protobuf info from new fields in the struct and the codgen process will add it for you and _should_ get it right.

Reply:
I renumbered them by hand because it felt weird to not have an identifier at the beginning of the payload. I will see what the codegen process makes of it when I let the computer do the work.

Reply:
afaik, it looks at the highest numbered field, adds one, and that's its starting point for numbering new fields, regardless of where they're positioned in the struct. i.e. It could still be at the top (where I agree an identifier belongs), but end up as a much larger number. But I'm basing that on what I've seen it do so far and not knowledge of the underlying implementation.

Reply:
You were right, and I amended my commit to let the computer do its thing.